CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.0.0
+
+**Release date:** 2023-07-03
+
+This is the first stable release of the controller.
```

Comment:
Very happy to take suggestions here. Do not have my best writers hat on today.

Reply:
```suggestion
This is the first stable release of the controller. From now on, this controller follows the [Flux 2 release cadence and support pledge](https://github.com/fluxcd/flux2/blob/main/docs/release/controllers.md).
```

Reply:
The release cadence will be here https://deploy-preview-1543--fluxcd.netlify.app/flux/releases/ Maybe we need to merge the website PR first. We could also mention SLSA provenance from https://deploy-preview-1530--fluxcd.netlify.app/flux/security/slsa-assessment/